### PR TITLE
Update projectlive.modules

### PR DESCRIPTION
**Pull request by andrewelamb**

### CODE REVIEW

**Comment by jaybee84** [comment]
Will wait to merge this in after the demo scheduled for today is done ...

**Comment by andrewelamb** [comment]
There were some issues with the publications module, I added one more commit.

**Comment by andrewelamb** [comment]
@jaybee84 I updated projectlive modules. This has the theme concatenation issue fix, and adds the latest updates such as the study table being filterable by initiative.

File: R/app_ui.R [hunk 1]
```diff
@@ -24,6 +24,11 @@ app_ui <- function() {
         projectlive.modules::summary_snapshot_module_ui("summary_snapshot_ui_1"),
```
Comment (jaybee84) [inline R/app_ui.R:24]: Re: theme concatenation:
I am still seeing themes joined together by `|`
Screen Shot 2021-03-05 at 3 56 07 PM

File: R/app_ui.R [hunk 2]
```diff
@@ -24,6 +24,11 @@ app_ui <- function() {
...
         icon = shiny::icon("chart-area")
       ),
+      shiny::tabPanel(
+        "Publications",
+        projectlive.modules::publication_status_module_ui("publication_status_ui_1"),
+        icon = shiny::icon("chart-area")
```
Comment (jaybee84) [inline R/app_ui.R:30]: The bottom plot is being overtaken by the legend
Screen Shot 2021-03-05 at 3 58 35 PM

**Review by jaybee84 — CHANGES_REQUESTED** [review]

File: R/app_ui.R (hunk 1 above)
Comment (andrewelamb) [reply R/app_ui.R:24]: This update doesn't have that fix, it was the one done right before Julie's demo.

File: R/app_ui.R (hunk 2 above)
Comment (andrewelamb) [reply R/app_ui.R:30]: We could either truncate this, or we should pick something else to use as fill colors.

File: R/app_ui.R (hunk 1 above)
Comment (jaybee84) [reply R/app_ui.R:24]: So should we update the renv.lock file for this PR to update the projectlive.module version? or would you like to start a new PR with all the updates?

Comment (andrewelamb) [reply R/app_ui.R:24]: Either is fine with me. The config files will need to be updated I think as well.

Comment (andrewelamb) [reply R/app_ui.R:24]: I guess my preference would be to accept this merge request and I'll open another branch to make the latest updates.

Comment (jaybee84) [reply R/app_ui.R:24]: just so we (mostly me) do not get confused as to which updates have been added, would you mind starting a PR against the current `update_projectlive.modules` to add in the updates? 
Then I can merge them into this branch and then merge the updated branch into the main demo branch.

**Pull request merged**